src/relay.py

```diff
@@ -99,7 +101,7 @@ def setRelayPinLow(relayPin):
 
 # *** Relay activate/deactivate/toggle ***
 
-
+# could possible combine the bottom two functions into one? and have activate level determine whether to set high or low? - DS
```

Comment:
I have done this in the `UnitTest-relay` branch, and tested it on the Unicon as well to ensure that it works fine. It can be merged into `develop` branch once it has been reviewed